generator/php/resources/templates/OAuthApiTest.mustache

```diff
@@ -1,82 +0,0 @@
-<?php
-namespace {{invokerPackage}};
-
-use {{invokerPackage}}\Api\OAuthApi;
-use {{invokerPackage}}\TokenAutoRefreshClient;
-use {{invokerPackage}}\ApiException;
-use {{invokerPackage}}\ObjectSerializer;
-use Jchook\AssertThrows\AssertThrows;
-use PHPUnit\Framework\TestCase;
-
-class OAuthApiTest extends TestCase
-{
-    use AssertThrows;
-
-    /**
-     * @var string
-     */
-    private $clientId;
-
-    /**
-     * @var string
-     */
-    private $clientSecret;
-
-    /**
-     * @var int
-     */
-    private $applicationId;
-
-    function __construct()
-    {
-        parent::__construct();
-        
-        $this->clientId = $_SERVER["TEST_CLIENT_ID"];
-        $this->clientSecret = $_SERVER["TEST_CLIENT_SECRET"];
-        $this->applicationId = $_SERVER["TEST_APPLICATION_ID"];
-    }
-
-    public function testSecretEnvironmentVariables()
-    {
-        $this->assertNotNull($this->clientId);
-        $this->assertNotNull($this->clientSecret);
-        $this->assertNotNull($this->applicationId);
-    }
-
-    public function testClientCredentialsGetTokenShouldSucceedWithValidCredentials()
-    {
-        // Arrange
-        $api = new OAuthApi();
-
-        // Act
-        $response = $api->getTokenWithHttpInfo('client_credentials', $this->clientId, $this->clientSecret);
-
-        // Assert
-        $this->assertEquals(200, $response[1]);
-        $this->assertNotNull($response[0]->getAccessToken());
-    }
-
-    public function testClientCredentialsGetTokenShouldFailWithInvalidCredentials()
```

Comment:
thinking out loud: could it make sense to add a test on `GatewayApiTest` to cover the behavior in case of invalid credentials?